### PR TITLE
EIP-3855: Review mode

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@alita-moore this for some reason didn't ping anyone

File: EIPS/eip-3855.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: PUSH0 instruction
 description: Introduce a new instruction which pushes the constant value 0 onto the stack
```
Comment (MicahZoltu) [inline EIPS/eip-3855.md:4]: Unrelated to this PR: We need to get the eips.ethereum.org site updated to render this field.

**Review by MicahZoltu — APPROVED** [review]

File: EIPS/eip-3855.md (hunk 1 above)
Comment (MicahZoltu) [reply EIPS/eip-3855.md:4]: I lied, it *is* included, just below the title rather than in the table.

Comment (axic) [reply EIPS/eip-3855.md:4]: Yeah, remember #3706 where that option was most liked? :)

**Pull request merged**